### PR TITLE
Add LFMP Students

### PR DESCRIPTION
**Pull request by PIYUSHgoyal16**

### CODE REVIEW

File: _pages/lfmp-2020-students.html [hunk 1]
```diff
@@ -20,6 +20,17 @@ <h4><b>Nidhi Jain</b></h4>
...
+
+    <div class="col-lg-3 centered">
+      <img class="img img-circle" src="../assets/images/abhik.jpeg" alt="Abhik Chakraborty">
+      <h4><b>Dipanshu Verma</b></h4>
+      <p><i>IPP scan (or virtual MF device) server (Scanner Application)</i></p>
+    </div>
+
+    <div class="col-lg-3 centered">
+      <img class="img img-circle" src="../assets/images/rishabh.jpeg" alt="Rishab Arya">
+      <h4><b>Dipanshu Verma</b></h4>
```
Comment (tillkamppeter) [inline _pages/lfmp-2020-students.html:32]: It needs Rishabh here.

Comment (PIYUSHgoyal16) [reply _pages/lfmp-2020-students.html:32]: Thanks, Till, corrected the typo.

File: _pages/lfmp-2020-students.html [hunk 2]
```diff
@@ -20,6 +20,17 @@ <h4><b>Nidhi Jain</b></h4>
       <h4><b>Dipanshu Verma</b></h4>
       <p><i>Wrapping proprietary printer drivers into a Printer Application</i></p>
     </div>
-   
+
+    <div class="col-lg-3 centered">
+      <img class="img img-circle" src="../assets/images/abhik.jpeg" alt="Abhik Chakraborty">
+      <h4><b>Dipanshu Verma</b></h4>
```
Comment (tillkamppeter) [inline _pages/lfmp-2020-students.html:26]: Here there is also the wrong name.

File: _pages/lfmp-2020-students.html (hunk 1 above)
Comment (tillkamppeter) [reply _pages/lfmp-2020-students.html:32]: I mean that you show the picture of Rishabh but the name displayed is Dipanshu.

**Pull request merged**